packages/reports/addon/components/report-actions/multiple-format-export.ts

```diff
@@ -67,7 +67,9 @@ export default class MultipleFormatExport extends ReportActionExport {
    * Href for google sheet export
    */
   get gsheetExportHref() {
-    return `/gsheet-export/report/${this.args.model.id}`;
+    const { compression } = this;
+
+    return `/gsheet-export/report?model=${compression.compress(this.args.model)}`;
```

Comment:
Can we have a test to check the value of the export and to deserialize it back to a report

```suggestion
    return `/gsheet-export/report?model=${compression.compressModel(this.args.model)}`;
```